gcp/api/server_new.py

```diff
@@ -94,8 +99,15 @@ def query_package(context,
   return bugs
 
 
-def affected_affects(version: str, affected: osv.AffectedVersions) -> bool:
+def affected_affects(name: str, version: str,
+                     affected: osv.AffectedVersions) -> bool:
   """Check if a given version is affected by the AffectedVersions entry."""
+  # Make sure the package name correctly matches this entity.
+  if affected.ecosystem != 'GIT' and name != affected.name:
+    return False
+  if affected.ecosystem == 'GIT' and osv.normalize_repo_package(name) != name:
```

Comment:
Is this correct? i.e. name or affected.name?

Reply:
Oh that was wrong - good catch